tools/common/Target.cs

```diff
@@ -557,6 +557,9 @@ public void GenerateMain (StringBuilder sb, ApplePlatform platform, Abi abi, str
 						GenerateIOSMain (sw, abi);
 						break;
 					case ApplePlatform.MacOSX:
+#if NET
+						GenerateIOSMain (sw, abi);
+#endif
```

Comment:
no `#else` ?

Reply:
🤦

Reply:
I looked at that too and after pondering for few minutes I came to the conclusion that it was intentional and the generated `xammac_setup` method is called somewhere... Well, perhaps a wrong conclusion on my part :-)